e2e-tests/development-runtime/cypress/integration/hot-reloading/page-queries.js

```diff
@@ -4,6 +4,10 @@ beforeEach(() => {
   cy.visit(`/page-query/`).waitForRouteChange()
 })
 
+after(() => {
+  cy.exec(`npm run reset`)
```

Comment:
Why not before too, like in the other case?

Reply:
I guess there is no reason to not to reset in `before` - I added `after` case when looking at `git status` what was left behind after running those tests locally 🤷‍♂️

Reply:
I also didn't want to spend too much time on "unrelated" stuff ;)

Reply:
oh and `npm run reset` only restore files that were changed with `npm run update --file X --some other flags` in tests - so this is mostly rather cheap and often can be no-op if there was no changes at all - so `before` should be mostly no-op. This stuff doesn't use `git` - it has its own changes log so it's actually quite fast

Reply:
:+1: